console/src/main/java/org/matrix/console/services/EventStreamService.java

```diff
@@ -640,12 +635,19 @@ private Notification buildNotification() {
                 Intent.FLAG_ACTIVITY_SINGLE_TOP);
 
         PendingIntent pi = PendingIntent.getActivity(this, 0, i, 0);
-
-        notification.setLatestEventInfo(this, getString(R.string.app_name),
-                "Listening for events",
-                pi);
-        notification.flags |= Notification.FLAG_NO_CLEAR;
```

Comment:
As @pvager noted on the comment on the old commit: `setOngoing(true)` replaces manually adding `FLAG_NO_CLEAR`.